src/libs/API.js

```diff
@@ -543,11 +543,12 @@ function ResendValidateCode(parameters) {
  * @param {Object} parameters
  * @param {String} parameters.password
  * @param {String} parameters.validateCode
+ * @param {String} parameters.accountID
  * @returns {Promise}
  */
 function SetPassword(parameters) {
     const commandName = 'SetPassword';
-    requireParameters(['email', 'password', 'validateCode'], parameters, commandName);
+    requireParameters(['accountID, password', 'validateCode'], parameters, commandName);
```

Comment:
There are some missing quotes here:
```suggestion
    requireParameters(['accountID', 'password', 'validateCode'], parameters, commandName);
```

Reply:
Ahh you're right! This was before when I was still adding the email (we changed it to use accountID so that it's more consistent with E.com and URL safe). I've changed the title of the PR, sorry for the confusion!